apache.conf

```diff
@@ -0,0 +1,7 @@
+<VirtualHost *:80>
+  DocumentRoot /phpipam
+  <Directory "/phpipam">
+    AllowOverride All
+    Require all granted
+  </Directory>
+</VirtualHost>
```

Comment:
Kept apache.conf simple since I would except most people host this behind an HTTPS terminating load balancer or are using it in an environment where they don't care if it's over HTTPS.